lldb/test/API/commands/memory/read/TestMemoryRead.py

```diff
@@ -41,6 +41,20 @@ def test_memory_read_c_string(self):
             " Consider increasing the maximum read length.",
         )
 
+    @skipIf(archs=no_match("^(riscv|aarch64).*"))
+    def test_riscv_decode_failure(self):
+        """Test the 'memory read' command with instruction format."""
+        self.build_run_stop()
+
+        # assume that 0xffffffff is invalid instruction in RISC-V and AArch64
+        # and decoding it will fail
+        # (lldb) memory read --format instruction `&my_insns[0]`
+        # warning: failed to decode instructions at 0x
```

Comment:
I don't think we need the example as the output we're matching isn't complex enough to need it.

Good when you do some regex magic though, so worth considering in tests generally.